ex2_bod_css.css

```diff
@@ -0,0 +1,31 @@
+.div1 {
```

Comment:
So, just for Windows Explorer support? Oh, dear... :stuck_out_tongue: That's just one of the reasons why Windows is never going to be developers' #1 choice for for a development environment any time soon... Any decent IDE / editor will properly display the extension by default, though, so my recommendation: skip the redundant suffix.